frontend/package.json

```diff
@@ -7,18 +7,23 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "antd": "^4.12.3",
+    "axios": "^0.21.1",
+    "babel-plugin-import": "^1.13.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.2",
+    "react-scripts": "^4.0.3",
+    "redux": "^4.0.5",
     "web-vitals": "^1.0.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "postinstall": "npm run build"
   },
+  "engines": {"node": "12.18.3", "npm": "6.14.6"},
```

Comment:
1. Should "postinstall" be "npm run build" or "yarn add"?
2. Should we add "yarn" to "engines"? (the version of npm and node is based on my package version)

Reply:
FYI: I used yarn to install packages for our frontend...